CHANGELOG.md

```diff
@@ -8,6 +8,22 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 
+The release adds logs support.
+Set `OTEL_LOGS_EXPORTER=otlp` to enable logs support.
+However, take notice that as for now
+the OpenTelemetry Go Logs API is not stable.
+
+### Added
+
+- Add logs support and `OTEL_LOGS_EXPORTER` environment variable.
+  `OTEL_LOGS_EXPORTER` accepts:
+  `none` - logs disabled,
+  `otlp` - OTLP gRPC exporter.
```

Comment:
This needs to be restructured to use markdown syntax for lists.

![20250207_092254](https://github.com/user-attachments/assets/edc5acef-05e8-43bd-aa8c-02bea03006e1)